eZ/Bundle/EzPublishIOBundle/Migration/MigrationHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * File containing the MigrationHandler class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishIOBundle\Migration;
+
+use eZ\Bundle\EzPublishIOBundle\ApiLoader\HandlerFactory;
+use Psr\Log\LoggerInterface;
+
+abstract class MigrationHandler implements MigrationHandlerInterface
+{
+    /** @var \eZ\Bundle\EzPublishIOBundle\ApiLoader\HandlerFactory */
+    protected $metadataHandlerFactory;
+
+    /** @var \eZ\Bundle\EzPublishIOBundle\ApiLoader\HandlerFactory */
+    protected $binarydataHandlerFactory;
+
+    /** @var \Psr\Log\LoggerInterface */
+    protected $logger;
```

Comment:
@bdunogier I could privatise the data handler properties and make final protected getters for them. OK?